pytest-embedded/pytest_embedded/dut.py

```diff
@@ -117,7 +118,19 @@ def expect(self, pattern, **kwargs) -> Match:  # noqa
             - `AnyStr`: if you're matching `pexpect.EOF` or `pexpect.TIMEOUT` to get all the current buffers.
             - `re.Match`: if matched given string.
         """
-        return self.pexpect_proc.expect(pattern, **kwargs)
+
+        not_matching: List[str] = kwargs.pop('not_matching', [])
+        index: Match = self.pexpect_proc.expect(pattern, **kwargs)
```

Comment:
```suggestion
        index: int = self.pexpect_proc.expect(pattern, **kwargs)
```

index should be an int?

Reply:
Yes, it returns an int, but I am just following the function annotation...
`def expect(self, pattern, **kwargs) -> Match:  # noqa`
Should we just change it to int, or is there some idea behind this?